NetKet/Machine/machine.hpp

```diff
@@ -71,16 +73,21 @@ class Machine : public AbstractMachine<T> {
   }
 
   void Init(const Hilbert &hilbert, const json &pars) {
+
     if (pars["Machine"]["Name"] == "RbmSpin") {
       m_ = Ptype(new RbmSpin<T>(hilbert, pars));
     } else if (pars["Machine"]["Name"] == "RbmMultival") {
       m_ = Ptype(new RbmMultival<T>(hilbert, pars));
+    } else if (pars["Machine"]["Name"] == "Jastrow") {
+      m_ = Ptype(new Jastrow<T>(hilbert, pars));
     }
   }
 
   void Init(const Graph &graph, const Hilbert &hilbert, const json &pars) {
     if (pars["Machine"]["Name"] == "RbmSpinSymm") {
       m_ = Ptype(new RbmSpinSymm<T>(graph, hilbert, pars));
+    } else if (pars["Machine"]["Name"] == "JastrowSymm") {
+        m_ = Ptype(new JastrowSymm<T>(graph,hilbert, pars));
```

Comment:
some consistent indentation here and there, `clang-tidy` is the solution, see `CONTRIBUTING`